tests/test_script.py

```diff
@@ -20,12 +20,12 @@ def get_output(args):
     return child.communicate()[0].decode("utf-8")
 
 
-def test_script_with_whitelist():
+def test_script_with_explicit_whitelist():
     assert call_vulture(['vulture/core.py', WHITELIST]) == 0
 
 
-def test_script_without_whitelist():
-    assert call_vulture(['vulture/core.py']) == 1
+def test_script_with_implicit_whitelist():
+    assert call_vulture(['vulture/core.py']) == 0
```

Comment:
There should also be `test_script_without_whitelist()` which uses `--exclude whitelists/stdlib.py`.